src/modules/class-wpml-elementor-form.php

```diff
@@ -16,7 +16,17 @@ public function get_items_field() {
 	 * @return array
 	 */
 	public function get_fields() {
-		return array( 'field_label', 'placeholder', 'field_html', 'acceptance_text', 'field_options' );
+		return [
+			'field_label',
+			'placeholder',
+			'field_html',
+			'acceptance_text',
+			'field_options',
+			'step_next_label',
+			'step_previous_label',
+			'previous_button',
+			'next_button',
+		];
```

Comment:
@strategio when you mix refactoring to fixes, it's hard to tell one from the other.